utils/python/CIME/utils.py

```diff
@@ -8,16 +8,13 @@
 import os
 import time
 import re
-if sys.version_info[0] == 2:
```

Comment:
pylint was flagging this and we only support major version 2 for the forseeable future.